### ISSUE TITLE
I want to add this at my project

### ISSUE THREAD

**Issue by danilosilvadev** [codler/Grid-Layout-Polyfill#6]
[Here](https://github.com/danilosilvadev/hackathon/tree/gh-pages) it is a simple html+css using grid page.

**Your example says:**

> Simply add the script after jQuery
> 
> <script src="grid-layout-polyfill.min.js"></script>

I didn't understand what you mean with that.

**Is it that in the head?**

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="grid-layout-polyfill.min.js"></script>
```

Please i'd like more info about how to use this in my project.

**Comment by JoshuaMcKelvey** [comment]
Yes, this is how you should get the library to load into your project.
